### ISSUE TITLE
Py37 breaks with new release

### ISSUE THREAD

**Issue by markhull** [borgbackup/borg#4090]
## Have you checked borgbackup docs, FAQ, and open Github issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

BUG

#### Your borg version (borg -V).
borg 1.1.7-1

#### Operating system (distribution) and version.
Arch Linux 4.18.10

#### Full borg commandline that lead to the problem (leave away excludes and passwords)
```
borg create							\
--verbose                       \
--list							\
--stats							\
--show-rc                       \
--compression lz4               \
::'{hostname}-{now}'            \
/mnt/newdata
```
## Describe the problem you're observing.
Installed new release via pacman (Arch Linux) today, immediately got the error listed below.

#### Can you reproduce the problem? If so, describe how. If not, describe troubleshooting steps you took before opening the issue.
Can reproduce it every time I run borg (target location doesn't matter).

#### Include any warning/errors/backtraces from the system logs
```
Mon Oct  1 16:10:07 CDT 2018 Starting backup

Traceback (most recent call last):
  File "/usr/bin/borg", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3112, in <module>
    @_call_aside
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3096, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 3125, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 578, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 895, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 781, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'msgpack-python!=0.5.0,!=0.5.1,!=0.5.2,!=0.5.3,!=0.5.4,!=0.5.5,<=0.5.6,>=0.4.6' distribution was not found and is required by borgbackup
```

**Comment by ThomasWaldmann** [comment]
you need python-msgpack, this is a distribution problem (and iirc happened before).

**Comment by markhull** [comment]
Thank you for your timely response. I'll get python-msgpack and hopefully get backing up again.

**Comment by ThomasWaldmann** [comment]
Talk to arch package maintainer, they need to fix that.

On October 2, 2018 10:57:30 PM GMT+02:00, hulles <notifications@github.com> wrote:
>Thank you for your timely response. I'll get python-msgpack and
>hopefully get backing up again.
>
>-- 
>You are receiving this because you modified the open/close state.
>Reply to this email directly or view it on GitHub:
>https://github.com/borgbackup/borg/issues/4090#issuecomment-426427493

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by eli-schwartz** [comment]
@markhull the root cause of this is #3891 being rejected for hysterical raisins.

Also, the issue title is erroneous, since I see absolutely nothing even remotely related to py37.

...

Anyway, yes, our hackish workaround for the pull request being rejected, was accidentally removed. The person who packaged the new version mistakenly thought this was fixed upstream, which is obviously foolish because why would upstream ever agree to fix it...

We released the fixed version re-adding our workaround, on the 3rd, so this should be long since fixed.
